templates/autoconfig.php.erb

```diff
@@ -1,5 +1,9 @@
 <?php
 $AUTOCONFIG = array(
+<% if !@admin_user.empty? -%>
+  "adminlogin"    => "<%= @admin_user %>",
+  "adminpass"     => "<%= @admin_pass %>",
+<% end -%>
   "dbtype"        => "<%= @db_type %>",
```

Comment:
This should be changed to `unless @admin_user.empty? and @admin_pass.empty?` to avoid an empty adminpass parameter.